pkg/ansible/playbook/setupNode.yml

```diff
@@ -6,4 +6,16 @@
     - name: modify permissions
       shell: chmod 755 avalanchego-installer.sh
     - name: call avalanche go install script
-      shell: ./avalanchego-installer.sh --ip static --rpc private --state-sync on --fuji
+      shell: ./avalanchego-installer.sh --ip static --rpc private --state-sync on --fuji --version {{ avalancheGoVersion }}
+    - name: get avalanche cli install script
+      shell: wget -nd -m https://raw.githubusercontent.com/ava-labs/avalanche-cli/main/scripts/install.sh
+    - name: modify permissions
+      shell: chmod 755 install.sh
+    - name: run install script
+      shell: ./install.sh -n
+    - name: create .avalanche-cli dir
+      shell: mkdir .avalanche-cli
+    - name: copy metrics config to cloud server
+      copy:
```

Comment:
can we just do `shell:  echo '{"MetricsEnabled}":false}' > /home/ubuntu/.avalanche-cli/config` instead of copying a local file (that also may not be present or configured in the expected way)

Reply:
I see. Anyway I believe copying a file from local machine is not needed in this case, unless there is a reason to keep local configuration? Also there may be other configuration in the file not expected by the server. Would like to know @dasconnor opinion on this one

Reply:
I'm ok copying the file, that way we might get some extra metrics haha